webgpu.h

```diff
@@ -497,6 +498,7 @@ typedef struct WGPUBindGroupLayoutEntry {
     WGPUTextureViewDimension viewDimension;
     WGPUTextureComponentType textureComponentType;
     WGPUTextureFormat storageTextureFormat;
+    uint64_t minBufferBindingSize;
```

Comment:
let's put it right after `hasDynamicOffset` since they both apply to buffers

Reply:
Done